package.json

```diff
@@ -120,6 +120,7 @@
     "lerna": "4.0.0",
     "lint-staged": "11.1.2",
     "make-dir-cli": "3.0.0",
+    "node-cleanup": "2.1.2",
```

Comment:
@Leeyoungone You are welcome and I'm glad the advice helped!
This diff is looking much better and it's mostly LGTM, but of course I have a nit about the forgotten `node-cleanup` dependency declaration that needs to be cleaned up (haha, what a great pun, I know).
TLDR: Remove this line from the diff and also make sure to do the same to the diff in the yarn.lock file

Reply:
lol the pun is so bad it's borderline good, I just got rid of the dependency so we should be good to go!!

Reply:
> lol the pun is so bad it's borderline good, I just got rid of the dependency so we should be good to go!!

@Leeyoungone Haha, thank you for taking care of it!